tests/rptest/test_many_partitions.yml

```diff
@@ -0,0 +1,9 @@
+# TODO work in progress getting clustered mode issues worked out
+partitions:
+  included:
+    - scale_tests/many_partitions_test.py
+
+  excluded:
+    - scale_tests/many_partitions_test.py::ManyPartitionsTest.test_many_partitions
+    - scale_tests/many_partitions_test.py::ManyPartitionsTest.test_many_partitions_compacted
+    - scale_tests/many_partitions_test.py::ManyPartitionsTest.test_omb
```

Comment:
Just for my own understanding, what does creating this yaml do? Does this become runnable as a schedule in the future if we want one? (though it's just not plugged in yet)

Reply:
With this YAML we can supply it to cdt-custom and create a job that runs the tiered test and all future tests that is to be added there, Excluded ones are already part of the scale_test pipeline